ibm_i/tests/test_sql_queries.py

```diff
@@ -0,0 +1,51 @@
+# (C) Datadog, Inc. 2021-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from datadog_checks.ibm_i.config_models import InstanceConfig
+from datadog_checks.ibm_i.config_models.instance import MessageQueueInfo
+from datadog_checks.ibm_i.queries import query_map
+
+
+def test_get_message_queue_info():
```

Comment:
Is it possible to split this test into 3? Or possibly parametrize the inputs/outputs with `@pytest.mark.parametrize`. Example: https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/tests/base/utils/db/test_util.py#L80-L151

Reply:
Right ! This is fixed, PTAL